### PR TITLE
Language update for attributes

### PR DESCRIPTION
**Pull request by minheragon12345**
# Description
Changed some minor language issue for custom attributes

## List of changes
- Jump Power
- Critical Chance
- Critical Multiplier

## Type of Change
- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: src/main/java/com/github/manasmods/manascore/attribute/ManasCoreAttributes.java [hunk 1]
```diff
@@ -19,7 +19,7 @@ public class ManasCoreAttributes {
     private static final DeferredRegister<Attribute> registry = DeferredRegister.create(ForgeRegistries.ATTRIBUTES, ManasCore.MOD_ID);
     public static final RegistryObject<Attribute> JUMP_POWER = registry.register("jump_power", () -> new RangedAttribute("manascore.attribute.jump_power.name", 0.42, 0, 800).setSyncable(true));
     public static final RegistryObject<Attribute> CRIT_CHANCE = registry.register("crit_chance", () -> new RangedAttribute("manascore.attribute.crit_chance.name", 0, 0, 100).setSyncable(true));
-    public static final RegistryObject<Attribute> CRIT_MULTIPLIER = registry.register("crit_multiplier", () -> new RangedAttribute("manascore.attribute.crit_multiplier.name", 1.5, 0, 420).setSyncable(true));
+    public static final RegistryObject<Attribute> CRIT_MULTIPLIER = registry.register("crit_multiplier", () -> new RangedAttribute("manascore.attribute.crit_multiplier.name", 1.5, -1.5, 420).setSyncable(true));
```
Comment (Charismara) [inline src/main/java/com/github/manasmods/manascore/attribute/ManasCoreAttributes.java:22]: why would u change the min value to `-1.5`?

**Review by Charismara — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/manasmods/manascore/attribute/ManasCoreAttributes.java (hunk 1 above)
Comment (minheragon12345) [reply src/main/java/com/github/manasmods/manascore/attribute/ManasCoreAttributes.java:22]: Isnt it `vanilla damage + (weapon damage * crit attribute)` ? it's `weapon damage * (1.5 + crit attribute)`
image
u said this and it does work like that ^

Comment (minheragon12345) [reply src/main/java/com/github/manasmods/manascore/attribute/ManasCoreAttributes.java:22]: therefore if the crit attribute is -1.5
the final damage will be `weapon damage * (1.5 - 1.5) = 0`

Comment (Charismara) [reply src/main/java/com/github/manasmods/manascore/attribute/ManasCoreAttributes.java:22]: it is `vanilla * attribute`. 
see here: https://github.com/ManasMods/ManasCore/blob/1.19.2/master/src/main/java/com/github/manasmods/manascore/attribute/ManasCoreAttributeHandler.java#L59

But the default amount of the attribute is 1.5 so by default a crit is 50% stronger. See here: https://github.com/ManasMods/ManasCore/blob/1.19.2/master/src/main/java/com/github/manasmods/manascore/attribute/ManasCoreAttributes.java#L22

**Review by Charismara — APPROVED** [review]

**Pull request merged**